Source/Core/InputCommon/ControllerEmu/ControlGroup/ModifySettingsButton.h

```diff
@@ -18,10 +18,11 @@ class ModifySettingsButton : public Buttons
 
   void AddInput(std::string button_name, bool toggle = false);
 
+  // Modifies the state
   void GetState();
```

Comment:
Same here, being weird as non `const` (and probably out of scope)

Reply:
I didn't write this method nor removed the const, I just think it's good to clarify that a function called GetState() does modify the state internally.

Reply:
Renamed to UpdateState() because the name GetState() made no sense here